Sniffs.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Sniffs;
+
+class Sniffs
+{
+    const DEFAULT_POINTS = 30000;
```

Comment:
These points are really low - is that intentional? FWIW the A->B jump happens at 1,000,000 points